<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>895</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="latit_inp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>60.603534</string>
   </property>
   <property name="placeholderText">
    <string>Input latitude</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Find it!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="longit_inp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>56.838306</string>
   </property>
   <property name="placeholderText">
    <string>Input longtude</string>
   </property>
  </widget>
  <widget class="QLabel" name="map_line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>1101</width>
     <height>791</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.010000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
